int32 safeWayPoint

---

bool success
